src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -49,12 +57,26 @@ public boolean dataTypeIsNotModifiable(final String typeName) {
 
   @Override
   public String getDateLiteral(final String isoDate) {
-    String literal = super.getDateLiteral(isoDate);
+    String literal;
```

Comment:
nit: just remove this variable. It is not really needed in the way that the implementation works now.

Reply:
done